common/lib/protocol-definitions/src/summary.ts

```diff
@@ -21,41 +21,89 @@ export interface ISummaryCommitter {
     date: string;
 }
 
+/**
+ *  Represents a leaf node from the Summary Tree.
```

Comment:
Technically this would not be a leaf if its type is tree, right?

Reply:
Removed leaf :)